WEB_first project/ASP.NET CORE Project#1/Queries/Users/GetUserRolesQuery.cs

```diff
@@ -0,0 +1,17 @@
+﻿using MediatR;
+using System.Collections.Generic;
+using ASP.NET_CORE_Project_1.Models;
+using global::ASP.NET_CORE_Project_1.Models;
+
+namespace ASP.NET_CORE_Project_1.Queries.Users
+{
+    public class GetUserRolesQuery : IRequest<IList<string>>
+    {
+        public ApplicationUser User { get; set; }
+
+        public GetUserRolesQuery(ApplicationUser user)
```

Comment:
Instead of passing User as parameter through all the app, use pattern Context that will incapsulater HttpContextAccessor to get current user https://learn.microsoft.com/en-us/aspnet/core/fundamentals/http-context?view=aspnetcore-8.0